tests/evm_backends/base_env.py

```diff
@@ -45,6 +45,7 @@ class BaseEnv:
     """
 
     INVALID_OPCODE_ERROR = "NotImplemented"  # must be implemented by subclasses
+    OUT_OF_GAS_ERROR = "NotImplemented"  # must be implemented by subclasses
```

Comment:
maybe these can be properties which raise NotImplementedError in the base class?

Reply:
I didn't think it was worth the extra lines of complexity (`@property`, `def`, `docstring`, `return`, `empty line`)